salt/kubectl-client-cert/init.sls

```diff
@@ -0,0 +1,28 @@
+include:
+  - crypto
+
+/etc/pki/kubectl-client-cert.key:
+  x509.private_key_managed:
+    - bits: 4096
+    - user: root
+    - group: root
+    - mode: 444
+    - require:
+      - sls:  crypto
+      - file: /etc/pki
+
+/etc/pki/kubectl-client-cert.crt:
+  x509.certificate_managed:
+    - ca_server: {{ salt['mine.get']('roles:ca', 'ca.crt', expr_form='grain').keys()[0] }}
+    - signing_policy: minion
+    - public_key: /etc/pki/kubectl-client-cert.key
+    - CN: CaaSP
+    - days_valid: {{ pillar['certificate_information']['days_valid']['certificate'] }}
+    - days_remaining: {{ pillar['certificate_information']['days_remaining']['certificate'] }}
+    - backup: True
+    - user: root
+    - group: root
+    - mode: 644
+    - require:
+      - sls:  crypto
+      - file: /etc/pki
```

Comment:
Yes, thank you. Updated. Also opened https://github.com/kubic-project/salt/pull/152 to match this.